staff/david-cano/frontend/arroz/Arroz.prototype.concat.js

```diff
@@ -0,0 +1,17 @@
+Arroz.prototype.concat = function () {
```

Comment:
great job, it would be great to add in test a case calling the function with more than one array in the arguments, i think it would be pass with no problem